src/service/relay.rs

```diff
@@ -75,3 +64,28 @@ impl Service for RelayService {
         }
     }
 }
+
+impl RelayService {
+    /// affirm target block
+    pub async fn affirm(&mut self, target: u64) -> BridgerResult<H256> {
+
+        match self.darwinia.should_relay(target).await {
+            Ok(true) => {
+                trace!("Trying to relay block {}...", target);
+                let parcel = self.shadow.parcel(target as usize).await?;
+
+                match self.darwinia.affirm(parcel).await {
+                    Ok(hash) => {
+                        info!("Summited proposal {:?}", hash);
+                        Ok(hash)
+                    },
+                    Err(err) => Err(err),
+                }
+            }
+            Ok(false) => {
+                Err(Error::Etc(format!("No need to affirm block {}.", target)))
```

Comment:
`Etc` is the error type of the crate `Etc`, we should use `Error::Bridger("".to_string())` here.